packages/expo-crypto/android/src/test/java/expo/modules/crypto/CryptoModuleTest.kt

```diff
@@ -26,7 +26,7 @@ class CryptoModuleTest {
     val encoding = "base64"
     for (algorithm in algorithms) {
       val promise = PromiseMock()
-      module.digestStringAsync(algorithm, testValue, hashMapOf("encoding" to "base64") as Map<String, Any>?, promise)
```

Comment:
1. IMHO, it should not be a nullable type as `CryptoModule#digestStringAsync` method does throw an NPE when null-options are passed
2. There is no need of using mutable `HashMap` type

Reply:
2. That's right, totally agree